### DISCUSSION TITLE
weekly live stream

### DISCUSSION THREAD

**Discussion by iason9** [EddieHubCommunity/support#432] [Ideas]
which day to do the live??

**Comment by eddiejaoude** [comment]
Monday

**Comment by eddiejaoude** [comment]
Tuesday

**Comment by eddiejaoude** [comment]
Wednesday

**Comment by eddiejaoude** [comment]
Thursday

**Comment by eddiejaoude** [comment]
Friday

**Comment by eddiejaoude** [comment]
Saturday

**Comment by eddiejaoude** [comment]
Sunday

**Comment by iason9** [comment]
I suggest that Eddie gets to vote two times 🤓

**Reply by eddiejaoude** [reply]
LOL I agree 😜

**Reply by iason9** [reply]
Nice live today, Shall i mark Sunday as the answer?

**Reply by eddiejaoude** [reply]
thank you @thejason1 🤓 - it is coming up to the Festive season soon, may be we select a day in the new year? what do you think?

**Reply by iason9** [reply]
Yeah that's a good idea so that for our next year goals will be a lot of open source / freelancing / and getting involved in our community even more, can't wait to color my dashboard with green box all over the place.. spoiler alert: i already started 🤓.
 
but I do wish a lot of improvement for the community and real collaborations because this is what makes an impact in each one of us and in a larger scale as well.

**Reply by eddiejaoude** [reply]
Starting now is a great idea 👍 

Yeah next year I really want to double down on the community and get into a regular scheduled routine 🤓

**Reply by iason9** [reply]
Yeah will support each other and the community
